bindings/sundials4py/arkode/arkode.cpp

```diff
@@ -0,0 +1,244 @@
+/*------------------------------------------------------------------------------
+ * Programmer(s): Cody J. Balos @ LLNL
+ *------------------------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2025-2025, Lawrence Livermore National Security,
+ * University of Maryland Baltimore County, and the SUNDIALS contributors.
+ * Copyright (c) 2013-2025, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * Copyright (c) 2002-2013, Lawrence Livermore National Security.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ *----------------------------------------------------------------------------*/
+
+#include "sundials4py.hpp"
+
+#include <sundials/sundials_core.hpp>
+#include <sundials/sundials_stepper.hpp>
+
+#include <arkode/arkode.hpp>
+#include <arkode/arkode_arkstep.hpp>
+#include <arkode/arkode_butcher.h>
+#include <arkode/arkode_ls.h>
+#include <arkode/arkode_mristep.hpp>
+#include <arkode/arkode_sprkstep.hpp>
+
+#include "arkode/arkode_impl.h"
+#include "arkode_usersupplied.hpp"
+#include "sundials_adjointcheckpointscheme_impl.h"
+
+namespace nb = nanobind;
+using namespace sundials::experimental;
+
+namespace sundials4py {
+
+// Forward declarations of functions defined in other translation units
+void bind_arkode_erkstep(nb::module_& m);
+void bind_arkode_arkstep(nb::module_& m);
+void bind_arkode_sprkstep(nb::module_& m);
+void bind_arkode_lsrkstep(nb::module_& m);
+void bind_arkode_mristep(nb::module_& m);
+void bind_arkode_forcingstep(nb::module_& m);
+void bind_arkode_splittingstep(nb::module_& m);
```

Comment:
These seem to be included by headers above so double checking if this is necessary.

Reply:
These are not included by any of the above headers.